core/src/main/java/org/apache/iceberg/util/ThreadPools.java

```diff
@@ -33,11 +33,9 @@ public class ThreadPools {
   private ThreadPools() {}
 
   public static final String WORKER_THREAD_POOL_SIZE_PROP =
-      SystemProperties.WORKER_THREAD_POOL_SIZE_PROP;
+      SystemConfigs.WORKER_THREAD_POOL_SIZE.propertyKey();
```

Comment:
Wondering should we deprecate this variable for SystemConfigs one?  Not sure any real use case to keep it around.

Reply:
Deprecated it.